lib/host.py

```diff
@@ -242,7 +242,7 @@ def xo_server_add(self, username, password, label=None, unregister_first=True):
                 'allowUnauthorized': 'true',
                 'label': label
             }
-        ).decode()
+        )
```

Comment:
Should this be done by `xo_cli` itself? There other users of `xo_cli` that seem to expect a `str` like here: https://github.com/xcp-ng/xcp-ng-tests/pull/295/files#diff-979b2af8ef5ed828b12e562799c4ae87c3eeeaadafa3ae0074ec6962b6a3f799R213